engine/src/main/java/graphql/nadel/engine/execution/StrategyUtil.java

```diff
@@ -159,18 +159,28 @@ public static RootExecutionResultNode mergeTrees(List<RootExecutionResultNode> r
     private static ExecutionResultNode combineResultNodes(List<ExecutionResultNode> resultNodes) {
         List<ExecutionResultNode> children = new ArrayList<>();
         Map<String, Object> completedValueMap = new LinkedHashMap<>();
+        // TODO: feature flag check, perhaps?
```

Comment:
I would get rid of this comment. If you do want to feature flag this, you can use `NadelExecutionHints` class. You would need to create a field there and a consumer (gateway) would need to supply the value of the hint with the `NadelExecutionInput` for every query. 
I wouldn't bother though, but up to you of course :)